docs/data/date-pickers/migration-lab/migration-lab.md

```diff
@@ -81,3 +81,60 @@ Which will transform the imports like this:
 
 Components of the Community Plan such as `<DatePicker />` can be imported from both `@mui/x-date-pickers-pro` and `@mui/x-date-pickers`.
 Only [date adapters](/x/react-date-pickers/getting-started/#setup) such as `AdapterDayjs` can only be imported from `@mui/x-date-pickers/[adapterName]`.
+
+### 3. Handle breaking changes introduced in alpha
+
```

Comment:
@joserodolfofreitas I'm wondering if we should introduce a small sentence explaining that those breaking changes have been introduce to improve coherence

Reply:
I would definitely vote for introducing some explanation text that breaking changes were introduced for a good reason. 🙌

Reply:
Thanks, just tried to clarify that those breaking changes occured between `v5.0.0-alpha.0` and `v5.0.0`